internal/cmd/gocdk/main_test.go

```diff
@@ -73,14 +69,28 @@ func TestFindModuleRoot(t *testing.T) {
 		if err := os.Mkdir(subdir, 0777); err != nil {
 			t.Fatal(err)
 		}
-		err = ioutil.WriteFile(filepath.Join(dir, "go.mod"), []byte("module example.com\n"), 0666)
-		if err != nil {
-			t.Fatal(err)
-		}
 
 		got, err := findModuleRoot(context.Background(), subdir)
 		if got != dir || err != nil {
 			t.Errorf("findModuleRoot(ctx, %q) = %q, %v; want %q, <nil>", dir, got, err, dir)
 		}
 	})
 }
+
+func newTestModule() (dir string, cleanup func(), _ error) {
```

Comment:
Why do you need the return types to be named here?

[I thought these are mainly useful for `defer`s, though I may be wrong]

Reply:
we don't *need* them to be named, I guess. I thought naming the cleanup function made it clearer what it was for.

Reply:
How about naming only cleanup? (the `_` name for the error doesn't seem useful anyway).  i looked through the style guide, it's ambivalent w.r.t. named parameters. If you feel naming cleanup will improve clarity, that sounds good

Reply:
It compile errors if I only name the function, but I can give the `string` a blank identifier?

Reply:
I see.

Don't feel strongly about this. We don't hame return values in similar cases elsewhere in the CDK (for example see `xrayserver.NewExporter`), but if you feel it helps clarify the function that's fine. Up to you.

In any case the function needs a documentation comment explaining the semantic meaning of parameters and return values.

Reply:
Ended up removing the names on the return values; seemed clear enough once the function was documented.